### PR TITLE
docs(README): mention license

### PR DESCRIPTION
**Pull request by TheUnderScorer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ func main() {
 > ```shell
```
Comment (JuroUhlar) [inline README.md:99]: Consider making the intro paragraph more consistent with the example Node Readme: https://github.com/fingerprintjs/fingerprintjs-pro-server-api-node-sdk

File: README.md [hunk 2]
```diff
@@ -99,7 +99,7 @@ func main() {
...
 > cd example && FINGERPRINT_API_KEY=SECRET_API_KEY VISITOR_ID=VISITOR_ID_EXAMPLE go run getVisits.go
```
Comment (JuroUhlar) [inline README.md:100]: `Requirements.` -> `Requirements` (no dot)

File: README.md [hunk 3]
```diff
@@ -99,7 +99,7 @@ func main() {
...
 > ```
```
Comment (JuroUhlar) [inline README.md:101]: Consider splitting [Installation & Usage](https://github.com/fingerprintjs/fingerprint-pro-server-api-go-sdk/blob/61d5a6a189d20a882acbecbcc8e20b07d39cc464/README.md#installation--usage) into "How to install" and "Getting started" just for consistency sake

File: README.md [hunk 4]
```diff
@@ -221,3 +221,8 @@ Class | Method | HTTP request | Description
 ## Author
 
 support@fingerprint.com
```
Comment (JuroUhlar) [inline README.md:223]: Consider adding a Support and feedback section consistent with the Node SDK: https://github.com/fingerprintjs/fingerprintjs-pro-server-api-node-sdk#support-and-feedback

File: README.md (hunk 1 above)
Comment (TheUnderScorer) [reply README.md:99]: Done!

File: README.md (hunk 2 above)
Comment (TheUnderScorer) [reply README.md:100]: Done!

File: README.md (hunk 3 above)
Comment (TheUnderScorer) [reply README.md:102]: In GO, usually you just import the library and then it is installed automatically when you build the app, so I don't think that splitting this section here will make sense

File: README.md (hunk 4 above)
Comment (TheUnderScorer) [reply README.md:224]: Done!

**Review by JuroUhlar — DISMISSED** [review]

File: README.md [hunk 5]
```diff
@@ -34,7 +35,7 @@ This Go package is automatically generated by the [Swagger Codegen](https://gith
 - Package version: 3.4.2
 - Build package: io.swagger.codegen.v3.generators.go.GoClientCodegen
 
-## Requirements.
+## Requirements
 
 Go Lang 1.17 or higher
```
Comment (JuroUhlar) [inline README.md:40]: @TheUnderScorer @ilfa Does it make sense to define some kind of support policy here?

Comment (TheUnderScorer) [reply README.md:40]: We can add a note that we'll support two latest major versions of GO. WDYT @ilfa ?

Comment (ilfa) [reply README.md:40]: We can say that we keep Microsoft support policy (2 latest major versions as you mentioned) and provide the [link](https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core) to it

Comment (TheUnderScorer) [reply README.md:40]: I think that you meant this for Dotnet @ilfa 😁. What about GO?

Comment (ilfa) [reply README.md:40]: Oh no!
But it is fair for a golang too, we just need [another URL](https://go.dev/doc/devel/release#:~:text=checkout%20goX.Y.Z-,Release%20Policy,until%20the%20Go%201.8%20release.)

Comment (TheUnderScorer) [reply README.md:40]: Done!

File: README.md [hunk 6]
```diff
@@ -34,10 +35,12 @@ This Go package is automatically generated by the [Swagger Codegen](https://gith
...
 
+We keep the [GO support policy](https://go.dev/doc/devel/release) and we support the two last major versions of GO.
```
Comment (JuroUhlar) [inline README.md:42]:
```suggestion
We keep the [Go support policy](https://go.dev/doc/devel/release) and support the last two major versions of Go.
```

File: template/README.mustache [hunk 7]
```diff
@@ -42,10 +43,12 @@ This Go package is automatically generated by the [Swagger Codegen](https://gith
 For more information, please visit [{{{infoUrl}}}]({{{infoUrl}}})
 {{/infoUrl}}
 
-## Requirements.
+## Requirements
 
 Go Lang 1.17 or higher
 
+We keep the [GO support policy](https://go.dev/doc/devel/release) and we support the two last major versions of GO.
```
Comment (JuroUhlar) [inline template/README.mustache:50]:
```suggestion
We keep the [Go support policy](https://go.dev/doc/devel/release) and support the last two major versions of Go.
```

**Pull request merged**